parsely/parselyandroid/doc/mainpage.dox

```diff
@@ -46,7 +46,7 @@ api key. This is usually best to do in the `MainActivity`'s `onCreate` method.
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ParselyTracker.sharedInstance("examplesite.com", this);
+        ParselyTracker.sharedInstance("example.com", this);
```

Comment:
Confused what you mean by that. You mean changing examplesite.com to example.com? I prefer example.com simply because that domain exists specifically to be used like that. examplesite.com could be a legit domain someday.

Reply:
I could have been more clear. To restate: updating the documentation contained in this repo is fine, but I believe there needs to be a new documentation page created in our standard integration documentation that describes the Android SDK. The ticket to that effect for the Swift SDK is https://github.com/Parsely/engineering/issues/2928. If you think this isn't a requirement for the sprint, just let us know.

Reply:
Don't think it's a requirement for the sprint, but I do plan on updating the README for this project so that it reflects what's inside. We don't have any parsely help docs about it right now, so I don't see a need to create them at the moment. The SDK is fairly advanced, so this doesn't need to be front and center right now. Maybe in the future when both SDKs are updated we can coordinate a refresh of the docs on that page.

Reply:
FYI we're planning on making an unlinked public docs page for the iOS SDK for ease of integration, see https://github.com/Parsely/web/pull/8925